app/src/main/java/edu/wisc/ece/pinpoint/pages/pins/PinViewFragment.java

```diff
@@ -194,4 +263,14 @@ private void deletePin(DialogInterface dialog, int buttonId) {
             navController.popBackStack();
         });
     }
+
+    private void setupCommentRecyclerView(View view, ArrayList<Comment> comments) {
```

Comment:
Most of this recycler view setup should be done in the `onViewCreated` method. Set an empty adapter as the adapter as a placeholder until the data is ready. Look at what Mark did in `FeedFragment` for the recycler view and adapter.